apps/backend/src/mutations/UserMutations.spec.ts

```diff
@@ -366,3 +366,54 @@ describe('updateUserByOidcSub', () => {
     expect((result as typeof dummyUser).department).toBe(dummyUser.department);
   });
 });
+
+describe('upsertUserByOidcSub', () => {

```

Comment:
Would it be worth adding a test for if an error if thrown if a new user is added with an existing oidc?